input/number/reports.md

```diff
@@ -0,0 +1,28 @@
+## This page should be used for casual reporting of issues.
```

Comment:
I don't want pages that are casual reports of issues. More of investigation into the current state of things that we can ultimately bubble up into solutions. So I'd delete this - Also, take the template currently being used in `<select>`. Ultimately I'd like to get @mfreed7 content into a similar reporting page of the current state of things into the repo. That said, there is valuable discussion occuring in the Google doc so I'm not sure if it's worth the time yet.

Reply:
I can work more closely to the template on the `select` page, and could rephrase, but the idea of a place where someone can very casually drop an item seems a good one to me.

It should be curated in that way. Simply as a research/notepad for people to make observations.

I agree with the deep-dive structure, so I think an `overview.md` might be good for this folder also, but this isn't intended to create that sort of space.

Do you agree that kind of space is something you might want?

Reply:
@fardarter Starting off I think I'd prefer to have a label of issues, eg: I can open issues and filter to input=number to see issues people are hitting (these could both be bugs or limitations people are hitting [eg: I can't style the spinners, each browser ships different parts, etc]). The area where what you're proposing becomes valuable, at least to me, is when we've determined that to create an improved input=number (based on diggind and taking into account the known bugs [as GH issues]) requires us to solve the `size` attribute issue which should then be tracked on the document as tasks that need to land to complete the improved control. Whether that be a new feature & spec or bug fixing, each control will be different. I also still recommend opening the issue on WHATWG as well but I think having a list of known number issues within this repo will be valuable to inform the document but also allow them to be referenced across repos (this is one thing I find valuable about issues, not just a notepad). Does that make sense?

**To hopefully clarify it fully, the flow would look like this:**
* Look through filed issues on our repo, as well as bug databases, usecases, rants, work arounds, etc
* Build out the solid list of unique use cases
* Identify missing platform features & known issues
* Scope the problem based on impact (eg: do we actually need to spec and implement all missing platform features in order to improve either the end user or developer experience)
* Begin determining how to solve that problem